Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1788,7 +1822,57 @@ ARGUMENTS:\n\
 \n\
     - ReplaceCore('C1CCC1CN','C1CCC1[*]',False) -> '[1*]CN'\n\
 \n";
-    python::def("ReplaceCore", replaceCore,
+    python::def("ReplaceCore",
```

Comment:
I'll change these to proper doc tests using the trick I used before.  Might as well do the whole Chemtransforms.  Might take a day or so.